content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -14,7 +14,7 @@ This page shows how to access clusters using the Kubernetes API.
 
 {{% capture steps %}}
 
-## Accessing the cluster API
+## Accessing the Kubernetes API
```

Comment:
@sb1975 . I reread this task page. The page appears to briefly describe several ways a client could communicate with a k8s cluster using the Kubernetes API Server: kubectl, curl or wget, or other programmatic clients. I _think_ the title is somewhat unclear. As far as I can tell, there is no mention of the [cluster-api](https://cluster-api.sigs.k8s.io/) on the task page . 
@tengqm , would it help to change the title (for example: Client Access to the Kubernetes API)? The heading change seems okay.